buildscripts/semantic-convention/templates/SemanticAttributes.scala.j2

```diff
@@ -122,208 +118,512 @@ object {{class}} {
   /**
    * The name of an event describing an exception.
    *
-   * <p>Typically an event with that name should not be manually created. Instead {@see
-   * org.typelevel.otel4s.trace.Span#recordException(Throwable)} should be used.
+   * <p>Typically an event with that name should not be manually created. Instead
+   * `org.typelevel.otel4s.trace.Span#recordException(Throwable)` should be used.
    */
   final val ExceptionEventName = "exception"
 
   /**
    * The name of the keyspace being accessed.
    *
-   * @deprecated this item has been removed as of 1.8.0 of the semantic conventions. Please use {@link SemanticAttributes#DbName} instead.
+   * @deprecated this item has been removed as of 1.8.0 of the semantic conventions. Please use [[SemanticAttributes.DbName]] instead.
    */
-  @deprecated("Use SemanticAttributes.DbName instead", "")
+  @deprecated("Use SemanticAttributes.DbName instead", "Semantic Conventions 1.8.0")
   val DbCassandraKeyspace = string("db.cassandra.keyspace")
 
   /**
    * The <a href="https://hbase.apache.org/book.html#_namespace">HBase namespace</a> being accessed.
    *
-   * @deprecated this item has been removed as of 1.8.0 of the semantic conventions. Please use {@link SemanticAttributes#DbName} instead.
+   * @deprecated this item has been removed as of 1.8.0 of the semantic conventions. Please use [[SemanticAttributes.DbName]] instead.
    */
-  @deprecated("Use SemanticAttributes.DbName instead", "")
+  @deprecated("Use SemanticAttributes.DbName instead", "Semantic Conventions 1.8.0")
   val DbHbaseNameSpace = string("db.hbase.namespace")
 
   /**
    * The size of the uncompressed request payload body after transport decoding. Not set if
    * transport encoding not used.
    *
-   * @deprecated  this item has been removed as of 1.13.0 of the semantic conventions. Please use {@link SemanticAttributes#HttpRequestContentLength} instead.
+   * @deprecated  this item has been removed as of 1.13.0 of the semantic conventions. Please use [[SemanticAttributes.HttpRequestContentLength]] instead.
    */
-  @deprecated("Use SemanticAttributes.HttpRequestContentLength instead", "")
+  @deprecated("Use SemanticAttributes.HttpRequestContentLength instead", "Semantic Conventions 1.13.0")
   val HttpRequestContentLengthUncompressed = long("http.request_content_length_uncompressed")
 
   /**
-   * @deprecated This item has been removed as of 1.13.0 of the semantic conventions. Please use {@link SemanticAttributes#HttpResponseContentLength} instead.
+   * @deprecated This item has been removed as of 1.13.0 of the semantic conventions. Please use [[SemanticAttributes.HttpResponseContentLength]] instead.
    */
-  @deprecated("Use SemanticAttributes.HttpResponseContentLength instead", "")
+  @deprecated("Use SemanticAttributes.HttpResponseContentLength instead", "Semantic Conventions 1.13.0")
```

Comment:
Is it worth mentioning in which version of the semantic conventions the attribute has been deprecated?

Reply:
In general, `since` refers to the version of the **library**. In contrast, I mention the version of a third-party semantic convention.

Reply:
This was previously discussed in:
- https://github.com/typelevel/otel4s/pull/231#discussion_r1213368040

Reply:
Thanks, I reverted this change.

Reply:
can we put an otel4s version in there rather than an empty string?

Reply:
I don't see reasons why we shouldn't.

Reply:
The `semconv` module wasn't published yet. Can we set `since = 0.3.0` everywhere?

Reply:
that sounds right.

Reply:
I've updated the deprecation annotation.